### DISCUSSION TITLE
How to monitor hosting instance ?

### DISCUSSION THREAD

**Discussion by ZucchiPfl** [spiral-project/ihatemoney#1087] [Q&A]
Hi,

First thank you for this project, it's really great to have an hosted alternative to the proprietary existing solutions!

I used the installation from the marketplace on alwaysdata, everything is working great. I just was wondering, how could I monitor how many projects, the size used for each project and their activities to do some cleaning on my hosted instance? I tried to have a look at the database but couldn't find what I was searching for in plain text.

I saw that it existed an issue (#969) about this but I'm not an expert and I don't know how to run python commands directly from my hosted instance.

+++,

**Comment by almet** [comment]
Hey, you can access the dashboard on `/dashboard` if you want. You just need to enable it beforehand. See the documentation about that : https://ihatemoney.readthedocs.io/en/latest/configuration.html#activate-admin-dashboard
